server/message_queue_service.py

```diff
@@ -0,0 +1,124 @@
+import asyncio
+import json
+from typing import Dict
+
+import aio_pika
+from aio_pika import DeliveryMode, ExchangeType
+from aio_pika.exceptions import ProbableAuthenticationError
+
+from .config import config
+from .core import Service
+from .decorators import with_logger
+
+
+@with_logger
+class MessageQueueService(Service):
+    def __init__(self) -> None:
+        """
+        Service handling connection to the message queue
+        and providing an interface to publish messages.
+        """
+        self._logger.info("Message queue service created.")
```

Comment:
TBH I'm not sure if all these creation messages are necessary. They kindof just clutter the log output and make it harder to see actual important messages like "missing geoip database" or something. I'd either remove it entirely or demote it to `DEBUG` level.

Reply:
(Same for rating serviceI